.changeset/smooth-goats-move.md

```diff
@@ -0,0 +1,5 @@
+---
+'@guardian/source-foundations': patch
+---
+
+change the default line-height for textsans to regular instead of loose
```

Comment:
Would we consider this to be [a breaking change](https://docs.google.com/document/d/1Vi7m35w46F72z6n4fucsxm6i28xbmn2iMfPGM7zWGz8/edit#heading=h.xoh3rjy9fctr) (i.e. a major version bump)?

Reply:
yep — I'm about to push a change to make this a breaking change after asking @sndrs this morning. I wasn't sure about how we categorise these changes previously

Reply:
I've swapped this back to a `patch` change, because we're going to decouple the change in line height and the Storybook changes because of a large diff